Fix the misspelled name of "March" in Ukrainian.

Submitted upstream as:

https://sourceforge.net/tracker/?func=detail&aid=3561330&group_id=10894&atid=310894

--- ../library/msgs/uk.msg	2011-04-26 11:14:03.000000000 -0400
+++ ../library/msgs/uk.msg	2012-08-24 12:52:15.000000000 -0400
@@ -34,5 +34,5 @@
         "\u0441\u0456\u0447\u043d\u044f"\
         "\u043b\u044e\u0442\u043e\u0433\u043e"\
-        "\u0431\u0435\u0440\u0435\u0436\u043d\u044f"\
+        "\u0431\u0435\u0440\u0435\u0437\u043d\u044f"\
         "\u043a\u0432\u0456\u0442\u043d\u044f"\
         "\u0442\u0440\u0430\u0432\u043d\u044f"\
